solidity/dashboard/src/components/coverage-pools/LearnMoreBanner.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import Banner from "../Banner"
+import * as Icons from "../Icons"
+import NavLink from "../NavLink"
+
+const LearnMoreBanner = ({ onClose }) => {
+  return (
+    <Banner className="banner banner--coverage-pools">
+      <Banner.CloseIcon onClick={onClose} />
+      <div className="flex row space-between full-center">
+        <Banner.Icon icon={Icons.CoveragePool} />
+        <Banner.Title className="h3 text-white">
+          Deposit KEEP in the coverage pool to
+          <br />
```

Comment:
Hmm i don't think using `<br />` tag here is a good practise based on [`<br>` documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns) (see **Accessibility concerns**)

Maybe the better idea would be to keep it as default in the div and and margin/padding on the right.

Reply:
I used a `<p>` tag. I think this is the best that we can do according to the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns).